ldms/src/test/Makefile.am

```diff
@@ -39,4 +39,29 @@ test_metric_SOURCES = test_metric.c
 test_metric_LDADD = -lldms
 test_metric_LDFLAGS = $(AM_LDFLAGS) -pthread -lm
 
-TESTS = $(check_PROGRAMS)
+EXTRA_DIST=ldms-run-static-tests.sh.in
+EXTRA_DIST+=ldms-static-test-list.sh.in
+EXTRA_DIST+=ldms-static-test-bypass.in
+
+CLEANFILES=ldms-run-static-tests.sh
+CLEANFILES+=ldms-static-test-list.sh
+CLEANFILES+=ldms-static-test-bypass 
```

Comment:
You should remove these EXTRA_DIST and CLEANFILES lines.

Since these files are known to autoconf because you declared them in AC_CONFIG_FILES, autoconf already knows to include them in the dist. Further these files should _not_ be removed at "make clean" time, because these files are generated at configure time, not at make time. These files should only be cleaned at "make distclean" time.

But again, autoconf handles both of those situations for us automatically because you named these files in AC_CONFIG_FILES.

Reply:
Ah, okay. Didn't realize that they weren't necessary/needed. I'll remove them